src/libcore/default.rs

```diff
@@ -115,6 +134,35 @@ pub trait Default: Sized {
     fn default() -> Self;
 }
 
+/// A helper function that, when imported, allows one to reduce duplication, by
+/// saying just `default()` instead of `Default::default()`, when the compile
+/// can infer the value type.
+///
+/// For example:
+/// ```
+/// #![feature(default_free_fn)]
+///
+/// use std::default::default;
+///
+/// #[derive(Default)]
+/// struct SomeOptions {
+///     foo: i32,
+///     bar: f32,
+/// }
+///
+/// fn main() {
+///     let options = SomeOptions { foo: 42, ..default() };
+/// }
+/// ```
+/// Notice that we import the free function `default()`, which will effectively
+/// just forward to the `Default::default()` method of the corresponding class,
```

Comment:
```suggestion
/// Notice that we import the free function `default()`, which
/// forwards to the `Default::default()` method of its inferred return type,
```

- "effectively" -- unnecessary and misleading, since it does literally forward to Default::default, not just in effect.
- "just" -- does not provide any meaning in this sentence.
- "class" -- not a thing in Rust.

Reply:
Replaced with "Free `default()` funtion forwards to the [`Default::default()`] method."

Reply:
Actually, removed it altogether.